tests/10-create.js

```diff
@@ -0,0 +1,286 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+
+import {generateMultibase, splitDid} from './helpers.js';
+import {
+  shouldBeDidResolverResponse,
+  shouldBeValidDid,
+  shouldErrorWithData,
+  shouldHaveDidDereferencingError,
+  shouldHaveDidResolutionError,
+  shouldHaveValidVersion
+} from './assertions.js';
+import chai from 'chai';
+import {filterByTag} from 'vc-api-test-suite-implementations';
+
+const should = chai.should();
+const headers = {
+  Accept: 'application/ld+json;profile="https://w3id.org/did-resolution"'
+};
+
+// default valid bs58 ed25519 did
+const did = 'did:key:z6MktKwz7Ge1Yxzr4JHavN33wiwa8y81QdcMRLXQsrH9T53b';
+const {match, nonMatch} = filterByTag({
+  property: 'didResolvers',
+  tags: ['Did-Key']
```

Comment:
Can we make the tags all lower kebab case? Feels cleaner. This can be done in a separate PR.

Reply:
This can be done, but will require changes to another library so better in a separate PR.